Resources/views/process/_sidebar_process_info.html.twig

```diff
@@ -24,21 +24,21 @@
                 <div class="mb-3">
                     <h3>{{ 'sidebar.client.employment'|trans }}</h3>
 
-                    {% if process.client.receivesPublicAid %}
+                    {% if process.client.receivesPublicAid is defined and process.client.receivesPublicAid %}
```

Comment:
This could be written as `{% if process.client.receivesPublicAid|default(false) %}` which may be easier to read …

Reply:
Clever